scripts/ci-helper.sh

```diff
@@ -8,86 +8,6 @@ set -euo pipefail
 # We don't want to get stuck in some kind of interactive pager
 export GIT_PAGER=cat
 
```

Comment:
I'm not sure I see the value between splitting what we're doing to build and test into two separate scripts. I think it'd be fine to change the name if that makes responsibilities more clear, but to me it communicates that we're going to use it for continuous integration, which includes building and testing.